network/dag/notifier.go

```diff
@@ -291,7 +314,15 @@ func (p *notifier) Notify(event Event) {
 	}
 
 	if err := p.notifyNow(event); err != nil {
-		p.retry(event)
+		if isUnrecoverable(err) {
+			log.Logger().
+				WithError(err).
+				WithField(core.LogFieldTransactionRef, event.Hash.String()).
+				WithField(core.LogFieldEventSubscriber, p.name).
+				Errorf("Notify failed")
```

Comment:
Errors already imply "failed". It is better to log the decision instead.

```go
log.Printf("DAG event %#x dropped: %s", event.Hash, err)
```

Reply:
this is consistent with the other error loggin. made some modifications.